android/versioned-abis/expoview-abi25_0_0/build.gradle

```diff
@@ -104,7 +103,7 @@ dependencies {
   api 'com.google.android.gms:play-services-vision:15.0.2'
   api 'com.google.android.gms:play-services-wallet:15.0.1' //may need 10.+
   api 'com.stripe:stripe-android:2.1.+'
-  api 'com.github.tipsi:CreditCardEntry:1.4.8.10'
+  api 'com.github.tipsi:CreditCardEntry:1.5.0'
```

Comment:
Is it reachable? I'm asking, cause I had to build it manually and provide via `.aar` file.

Reply:
Yes, it's fetched via `jitpack.io`. BTW it was released 17 days ago, so you may have had to add AAR manually.